Kernel/Thread.cpp

```diff
@@ -853,8 +855,11 @@ DispatchSignalResult Thread::dispatch_signal(u8 signal)
         // Align the stack to 16 bytes.
         // Note that we push 176 bytes (8 * 22) on to the stack,
         // so we need to account for this here.
-        FlatPtr stack_alignment = (*stack - 112) % 16;
-        *stack -= stack_alignment;
+        // 22 % 2 = 0, so we dont need to take any thing that into consideration
+        // for the alignment.
+        // We also are not allowed to touch the threads red-zone of 128 bytes
```

Comment:
Typo: threads -> thread's

Reply:
But the thread isn't

or am I missing something

Reply:
Ok looking it up, apperantly english always uses a apostrophe for the posessive (or genetive) s